SDK/ViewControllers/CalendarItemViewController.swift

```diff
@@ -90,8 +90,10 @@ final class CalendarItemViewController: UIViewController {
     func loadICS(){
         apiClient.execute(query: CalendarQuery(url: icsURL), completion: { result in
             switch result {
-            case let .success(calendar):
-                self.calendar = calendar
+            case let .success(calendar):                
+                let upcomingEvents = calendar.events.filter({ $0.startDate > Date() })
+                let upcomingCalendar = ICalendar(events: upcomingEvents)
+                self.calendar = upcomingCalendar
```

Comment:
Does this mean we don't show any past events anymore? This is not the way we should handle this. Instead, after loading the events we should "jump" to the next upcoming event in the list.

Reply:
Change is up